<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoeDialog</class>
 <widget class="QDialog" name="SoeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输出总时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTime"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>触发方式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>使能</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>变位时刻(ms)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>闭合时间(ms)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>断开时间(ms)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>变位次数</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditChange_1"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEditClose_1"/>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="lineEdit_1"/>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="lineEditCount_1"/>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditChange_2"/>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEditClose_2"/>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="lineEditCount_2"/>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditChange_3"/>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEditClose_3"/>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="lineEditCount_3"/>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEditChange_4"/>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="lineEditClose_4"/>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="lineEditCount_4"/>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEditChange_5"/>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="lineEditClose_5"/>
   </item>
   <item row="7" column="3">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="lineEditCount_5"/>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="checkBox_6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEditChange_6"/>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="lineEditClose_6"/>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="8" column="4">
    <widget class="QLineEdit" name="lineEditCount_6"/>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="checkBox_7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="lineEditChange_7"/>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="lineEditClose_7"/>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="9" column="4">
    <widget class="QLineEdit" name="lineEditCount_7"/>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="checkBox_8">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="lineEditChange_8"/>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="lineEditClose_8"/>
   </item>
   <item row="10" column="3">
    <widget class="QLineEdit" name="lineEdit_8"/>
   </item>
   <item row="10" column="4">
    <widget class="QLineEdit" name="lineEditCount_8"/>
   </item>
  </layout>
  <action name="action">
   <property name="text">
    <string>开始实验</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
